etc/k8s/deployment.yaml

```diff
@@ -37,7 +39,7 @@ spec:
         imagePullPolicy: Always
         resources:
           limits:
-            cpu: "1"
```

Comment:
It's a good practice to define requests and limits. Requests is regarding pod to node allocation,  whereas limits is about maximum value of resource pod can reach. Be aware that define those limits need to happen after some load test with a monitoring tool. Take a look at prometheus-operator in kubernetes, it enables metrics a lot of resource metrics to help you to monitor your cluster and services. If you are not comfortable in do that now, leave only the requests specified, it's better than have your service in starvation due to lack of resources.

Other point is: this king of services used to consume all available CPU. So, if you don't specify any quota, it'll drain all cores available. Moreover, if you put any CPU limits different than an integer value, you'll suffer with high variance on execution time. This is because of the cpu time sharing when defining a fraction of a core, such as "100m". In general, if you want to ensure execution time of a service with that characteristic, put the CPU requests and limits equals, and use an integer number.